challenges/unit-testing/katas-tdd/calculator/calculator.test.js

```diff
@@ -0,0 +1,53 @@
+const add = require("./calculator.js");
+
+test("throw an error if num is not a string", () => {
+  const target = "Parameter must be a string.";
+  expect(() => {
+    add(9);
+  }).toThrow(target);
+});
+
+test("return 0 if param is empty", () => {
+  const str = "";
+  const currentInput = add(str);
+  const targetOutput = 0;
+  expect(currentInput).toEqual(targetOutput);
+});
+
+test("return number if param length is 1", () => {
+  const str = "9";
+  const currentInput = add(str);
+  const targetOutput = 9;
+  expect(currentInput).toEqual(targetOutput);
+});
+
+test("return sum of numbers if length of param is greater than 1", () => {
+  const str = "9, 1, 9, 1, 9, 1";
+  const currentInput = add(str);
+  const targetOutput = 30;
+  expect(currentInput).toEqual(targetOutput);
+});
+
+test("ignore numbers greater than 1000", () => {
+  const str = "200000, 1";
+  const currentInput = add(str);
+  const targetOutput = 1;
+  expect(currentInput).toEqual(targetOutput);
+});
+
+test("throw an error if numbers contain negative values", () => {
+  const targetOutput = "negatives not allowed: -3";
+  expect(() => {
+    const str = "200000, 1, -3";
+    add(str);
+  }).toThrow(targetOutput);
+});
+
+
+test("throw and error and display if multiple negative values", () => {
+    const targetOutput = "negatives not allowed: -3, -4";
+    expect(() => {
+      const str = "200000, 1, -3, -4";
+      add(str);
+    }).toThrow(targetOutput);
+  });
```

Comment:
could you please add tests where the input consists only of a single negative value, or a single large value?

Reply:
Done!